easybuild/easyconfigs/a/AMBER/AMBER-9.0.0-ictce-5.3.0.eb

```diff
@@ -0,0 +1,41 @@
+#
+# author: Marios Constantinou (University of Cyprus)
+#
+
+easyblock = 'MakeCp'
```

Comment:
this should go too, we should use the easyblock from https://github.com/hpcugent/easybuild-easyblocks/pull/491